lntest/itest/lnd_test_list_on_test.go

```diff
@@ -447,4 +447,16 @@ var allTestCases = []*testCase{
 		name: "open channel fee policy",
 		test: testOpenChannelUpdateFeePolicy,
 	},
+	{
```

Comment:
Since all three tests are small and share similar code, how about we merge them into one test, something like `neutrino rpc test`?

Reply:
Sounds reasonable.

Reply:
Seems not addressed?

Reply:
Will address this shortly. Just realised I updated the pr from my new computer and some code were missing. Sorry about that.